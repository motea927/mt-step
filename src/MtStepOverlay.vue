<template>
  <div class="mt-step-overlay"></div>
</template>

<script setup lang="ts"></script>

<style lang="scss">
.mt-step-overlay {
  background: #000;
  position: fixed;
  inset: 0;
  display: block;
  width: 100%;
  height: 100%;
  opacity: 0.75;
  z-index: var(--z-index-overlay);
}
</style>
