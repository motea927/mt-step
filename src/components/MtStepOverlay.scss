.mt-step-overlay {
  background: #000;
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  display: block;
  width: 100%;
  height: 100%;
  opacity: 0.75;
  z-index: var(--z-index-overlay);
}
