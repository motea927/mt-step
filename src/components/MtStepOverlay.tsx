import { defineComponent } from 'vue-demi'

import '@/components/MtStepOverlay.scss'

const MtStepOverlay = defineComponent({
  setup() {
    return () => <div class="mt-step-overlay"></div>
  },
})

export default MtStepOverlay
