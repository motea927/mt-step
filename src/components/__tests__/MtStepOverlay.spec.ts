import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import MtStepOverlay from '@/components/MtStepOverlay'

describe('MtStepOverlay.vue', () => {
  it('Renders correctly', () => {
    const wrapper = mount(MtStepOverlay)
    expect(wrapper.element).toMatchSnapshot()
  })
})
