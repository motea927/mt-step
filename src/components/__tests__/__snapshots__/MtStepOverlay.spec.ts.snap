// Vitest Snapshot v1

exports[`MtStepOverlay.vue > Renders correctly 1`] = `
<div
  class="mt-step-overlay"
/>
`;
